lib/linguist/languages.yml

```diff
@@ -8116,10 +8116,16 @@ Zimpl:
   extensions:
   - ".zimpl"
   - ".zmpl"
-  - ".zpl"
```

Comment:
Don't do this. We never remove support for an extension. To ensure your language is correctly identified, you will need to add a heuristic to differentiate the two languages.